dstyle.dd

```diff
@@ -391,5 +391,6 @@ $(P
 
 Macros:
     TITLE=The D Style
-    LISTSECTION=$(H4 $1) $(UL $+)
+    LISTSECTION=$(H4 $(LNAME2 $2, $1)) $(UL $(SKIPFIRST $+))
```

Comment:
I would prefer `LISTSECTION=$(H4 $(LNAME2 $1, $2)) $(UL $(SKIPFIRST $+))` to be consistent with `LNAME2`.